packages/p2p-v2/src/pages/orders/screens/OrdersChat/OrdersChat.scss

```diff
@@ -0,0 +1,10 @@
+.p2p-v2-orders-chat {
+    border-radius: 4px;
+    border: 8px solid var(--general-section-1);
```

Comment:
use the hard coded value instead here

Reply:
i think we can leave the variables here, since it would be the same from deriv-com ui themes and also it would be difficult to replace if its hardcoded.

Reply:
so all will be the same confirmed?

Reply:
that's what i got from deriv-com/ui team. but even if not, it would be easier to replace the variables than replacing the hardcoded values.

Reply:
hmmmm not necessarily, its easier to track what colour code is attached to the variable when they come out with the new colour scheme with all the variables 👀 , they might not have the same name so you would have to look for the colour code tied to your current variable, then search which variable in the new deriv-com/ui, has the correct one

Reply:
We should surely follow a standard theme, in the codebase certain elements should not be hard-coded to maintain flexibility like endpoint APIs, constants, magic numbers, styling, theming and many more.

let's raise this to the deriv-com/ui team and ask for a solutions 🙏🏼